dashboard/modules/metrics/metrics_head.py

```diff
@@ -157,7 +157,7 @@ async def prometheus_health(self, req):
 
                 text = await resp.text()
                 # Basic sanity check of prometheus health check schema
-                if "Prometheus" not in text:
+                if "Prometheus" not in text or "OK" not in text:
```

Comment:
can you add a comment to the code that explains this change?

Reply:
@rkooo567 Added! Let me know if you have any other questions.